### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by sculpt0r**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,18 +27,18 @@
 	"homepage": "https://github.com/ckeditor/ckeditor4-integrations-common#readme",
 	"browserslist": "> 0.25%, not dead",
 	"devDependencies": {
-		"@babel/core": "^7.11.6",
-		"@babel/preset-env": "^7.14.1",
+		"@babel/core": "^7.17.10",
```
Comment (KarolDawidziuk) [inline package.json]:
```suggestion
		"@babel/core": "^7.18.2",
```
There is a newer version of `babel/core`

File: package.json [hunk 2]
```diff
@@ -27,18 +27,18 @@
...
+		"@babel/preset-env": "^7.17.10",
 		"@rollup/plugin-babel": "^5.3.0",
-		"babel-loader": "^8.1.0",
+		"babel-loader": "^8.2.5",
 		"chai": "^4.2.0",
-		"eslint": "^8.10.0",
+		"eslint": "^8.14.0",
```
Comment (KarolDawidziuk) [inline package.json]:
```suggestion
		"eslint": "^8.16.0",
```
There is a newer version of `eslint`.

File: package.json [hunk 3]
```diff
@@ -27,18 +27,18 @@
...
 		"eslint-config-ckeditor5": "^3.1.0",
 		"istanbul-instrumenter-loader": "^3.0.1",
-		"karma": "^6.3.17",
+		"karma": "^6.3.19",
```
Comment (KarolDawidziuk) [inline package.json]:
```suggestion
		"karma": "^6.3.2",
```
There is a newer version of `karma`.

File: package.json [hunk 4]
```diff
@@ -47,7 +47,7 @@
 		"karma-sinon": "^1.0.5",
 		"karma-webpack": "^4.0.2",
 		"mocha": "^9.2.1",
-		"rollup": "^2.47.0",
+		"rollup": "^2.72.0",
```
Comment (KarolDawidziuk) [inline package.json]:
```suggestion
		"rollup": "^2.75.3",
```

There is a newer version of `rollup`.

**Review by KarolDawidziuk — CHANGES_REQUESTED** [review]
Hiello @sculpt0r and sorry for waiting so long for this review. 🙏 

Overall, everything looks good but since this PR was created some of the libraries have been updated. 
Please see my inline comments.

Also, let's add the same version of node(12) to the `nvmrc` file as used in `Travis`.

**Comment by sculpt0r** [comment]
@KarolDawidziuk - ready for another review :) All last-mentioned deps are bumped.

**Comment by sculpt0r** [comment]
Forgot to change `.nvmrc`: so now it uses version `12` - the same travis does.

**Review by KarolDawidziuk — APPROVED** [review]
Thank you @sculpt0r for the updates.

LGTM! 🎉 
I just update `eslint` because two days ago there was a new version.

**Pull request merged**